src/main/java/org/rumbledb/expressions/flowr/Clause.java

```diff
@@ -27,13 +27,13 @@
  * GRAMMAR:flowrExpression
  */
 
-public abstract class FlworClause extends Node {
+public abstract class Clause extends Node {
```

Comment:
I was initially reserved about using 'Clause' name instead of FlworClause with the worry of this keyword having a different meaning in some niche part of the codebase. However, after seeing that visitSwitchCaseClause is refactored out ('SwitchCaseClause' lives effectively only in the grammar now), I am more confident in this change.

Reply:
Indeed the use of "clause" for switch case is a syntactic detail that disappears in the expression tree.